src/sync/commits.ts

```diff
@@ -37,10 +36,6 @@ export default async (github: GitHubAPI, repository: Repository, cursor?: string
 		commitsData = await getCommits(true);
 	} catch (err) {
 
-		if (!await booleanFlag(BooleanFlags.RETRY_WITHOUT_CHANGED_FILES, false)) {
-			throw err;
-		}
```

Comment:
The FF is currently enabled on all environments, which means we can just remove these three lines.

Reply:
Do we have any tests from where we need to remove it?

Reply:
no, those were the wild west days... :(

Reply:
no, those were the wild west days... :(